### PR TITLE
Issue/20 install package

### PR DESCRIPTION
**Pull request by sustefil**
Rudimentary install package done. Usage now possible through an entry-point (CLI command).

Basic installation instructions provided:

https://github.com/csaf-tools/CVRF-CSAF-Converter/blob/issue/20-install-package/README.md#installation-with-venv

Points to discuss :

**1)** How to distribute the config file? For now it's inside the package, shall we put it inside `/etc/`, `/usr/local/share` or some other path?

See: https://github.com/csaf-tools/CVRF-CSAF-Converter/blob/issue/20-install-package/cvrf2csaf/common/utils.py#L12

**2)** Usage through CLI command is now possible, shall it be also possible to use it as a python module? (e.g. import cvrf2csaf; cvrf2csaf.convert(...))

### CODE REVIEW

File: cvrf2csaf/common/utils.py [hunk 1]
```diff
@@ -1,20 +1,27 @@
...
 def get_config_from_file():
     """Loads configuration file. Parts of it can be overwritten by CLI arguments."""
     config = dict()
     try:
-        f = open('./config.yaml', 'r')
+        # TODO: Workaround for now, config file placement is to be discussed
+        req = pkg_resources.Requirement.parse('cvrf2csaf')
+        path_to_conf = pkg_resources.resource_filename(req, 'cvrf2csaf/config/config.yaml')
+
+        f = open(path_to_conf, 'r')
```
Comment (sthagen) [inline cvrf2csaf/common/utils.py:16]: context manager?

**Review by sthagen — APPROVED** [review]
LGTM.

File: cvrf2csaf/common/utils.py (hunk 1 above)
Comment (sustefil) [reply cvrf2csaf/common/utils.py]: Absolutely, done

File: cvrf2csaf/common/utils.py [hunk 2]
```diff
@@ -1,19 +1,24 @@
...
+    """ Loads configuration file. Parts of it can be overwritten by CLI arguments. """
     config = dict()
     try:
-        f = open('./config.yaml', 'r')
-        config = yaml.safe_load(f)
+        # TODO: Workaround for now, config file placement is to be discussed
+        req = pkg_resources.Requirement.parse('cvrf2csaf')
+        path_to_conf = pkg_resources.resource_filename(req, 'cvrf2csaf/config/config.yaml')
+        with open(path_to_conf, 'r') as f:
+            config = yaml.safe_load(f)
```
Comment (sthagen) [inline cvrf2csaf/common/utils.py:16]: Nit: a single line body of `return yaml.safe_load(f)` should be enough though ...

**Review by sthagen — APPROVED** [review]
LGTM and thanks for considering my feedback.

File: cvrf2csaf/common/utils.py (hunk 2 above)
Comment (sustefil) [reply cvrf2csaf/common/utils.py:16]: For the sake of explicitness, let's leave it like this

**Comment by sustefil** [comment]
Hi @tschmidtb51, 

have a look at the points to discuss. I see them the lowest-prio for now, but just to have it clear. Thanks

**Pull request merged**